entity/templates/src/main/webapp/app/_entity-controller.js

```diff
@@ -45,23 +45,25 @@ angular.module('<%=angularAppName%>')
 
         $scope.delete = function (id) {
             <%= entityClass %>.get({id: id}, function(result) {
-                $scope.<%= entityInstance %> = result;
-                $('#delete<%= entityClass %>Confirmation').modal('show');
-            });
-        };
 
-        $scope.confirmDelete = function (id) {
-            <%= entityClass %>.delete({id: id},
-                function () {
-                    <%_ if (pagination != 'infinite-scroll') { _%>
-                    $scope.loadAll();
-                    <%_ } else { _%>
-                    $scope.reset();
-                    <%_ } _%>
-                    $('#delete<%= entityClass %>Confirmation').modal('hide');
-                    $scope.clear();
+                var modalInstance = $modal.open({
```

Comment:
Ya initially thought of doing that but then I realised that it might be a
bit dangerous as someone might hit a cached link or something and hit enter
to do a delete, so personally i wouldn't want delete action to be routes
On 31 Oct 2015 18:55, "Zsombor" <notifications@github.com> wrote:

> In entity/templates/src/main/webapp/app/_entity-controller.js
> <https://github.com/jhipster/generator-jhipster/pull/2255#discussion_r43570741>
> :
>
> >
> > -        $scope.confirmDelete = function (id) {
> > -            <%= entityClass %>.delete({id: id},
> > -                function () {
> > -                    <%_ if (pagination != 'infinite-scroll') { _%>
> > -                    $scope.loadAll();
> > -                    <%_ } else { _%>
> > -                    $scope.reset();
> > -                    <%_ } _%>
> > -                    $('#delete<%= entityClass %>Confirmation').modal('hide');
> > -                    $scope.clear();
> > +                var modalInstance = $modal.open({
>
> You can do the same, as you did it in the user-management screen :
> The dialog could be opened from the ui-router code, so it will have a nice
> url as well : /myentity/54/delete [image: :wink:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/2255/files#r43570741>
> .
>